packages/business-rules/src/schemas/householder-appeal/appeal-schema/eligibility/householder-planning-persmission/householder-planning-permission-validation.js

```diff
@@ -0,0 +1,7 @@
+const booleanValidation = require('../../../generic-validators/boolean-validation');
+
+const householderPlanningPermissionValidation = () => {
+	return booleanValidation();
```

Comment:
If we could use booleanValidation directly in the index.js it would reduce a lot of code. This kind of use is very redundant.